tests/formats/pdb.cpp

```diff
@@ -246,51 +287,34 @@ TEST_CASE("Read files in PDB format") {
         REQUIRE(frame.size() == 2223);
     }
 
-    SECTION("Read file generated by Crystal Maker") {
+    SECTION("File generated by Crystal Maker") {
         auto file = Trajectory("data/pdb/crystal-maker.pdb");
         REQUIRE(file.nsteps() == 1);
 
         auto frame = file.read();
         REQUIRE(frame.size() == 8);
     }
 
-    SECTION("Read Protein Residues") {
-        auto frame = Trajectory("data/pdb/hemo.pdb").read();
-        auto topology = frame.topology();
-
-        CHECK(!topology.are_linked(topology.residue(2), topology.residue(3)));
-        CHECK( topology.are_linked(topology.residue(3), topology.residue(4)));
-        CHECK(!topology.are_linked(topology.residue(3), topology.residue(5)));
-        CHECK(topology.bonds().size() == 482);
+    SECTION("Short CRYST1 record") {
+        auto file = Trajectory("data/pdb/short-cryst1.pdb");
+        auto frame = file.read();
     }
 
-    SECTION("Read Nucleic Residues") {
-        auto frame = Trajectory("data/pdb/2hkb.pdb").read();
-        auto topology = frame.topology();
+    SECTION("Short ATOM record") {
+        auto file = Trajectory("data/pdb/short-atom.pdb");
+        auto frame = file.read();
+        REQUIRE(frame.size() == 9);
 
-        CHECK(topology.are_linked(topology.residue(3), topology.residue(4)));
-        CHECK(!topology.are_linked(topology.residue(3), topology.residue(5)));
-        CHECK(topology.bonds().size() == 815);
-    }
+        CHECK(frame[0].name() == "O");
+        CHECK(frame[5].name() == "H");
+        CHECK(frame[0].type() == "O");
+        CHECK(frame[5].type() == "H");
 
-    SECTION("Read atomic insertion codes") {
-        auto frame = Trajectory("data/pdb/insertion-code.pdb").read();
-        auto& topology = frame.topology();
-
-        CHECK(topology.residue_for_atom(0)->get("insertion_code")->as_string() == "a");
-        CHECK(topology.residue_for_atom(1)->get("insertion_code")->as_string() == "c");
-        CHECK(topology.residue_for_atom(2)->get("insertion_code")->as_string() == "x");
-        CHECK_FALSE(frame[3].get("insertion_code"));
+        CHECK(approx_eq(frame.positions()[0], {0.417, 8.303, 11.737}, 1e-5));
+        CHECK(approx_eq(frame.positions()[5], {8.922, 9.426, 5.320}, 1e-5));
     }
 }
 
-TEST_CASE("Problematic PDB files") {
-    auto file = Trajectory("data/pdb/bad/atomid.pdb");
```

Comment:
Is this test no longer valid?

Reply:
The file moved to `data/pdb/atom-id-0.pdb`, and is still tested